### ISSUE TITLE
Vector layer renderMode: 'image' makes Chrome ver 69 crash

### ISSUE THREAD

**Issue by AnthonNilsson** [openlayers/openlayers#8600]
After the now latest chrome update (version 69.0.3497.81) renderMode: 'image' on a vector layer will have chrome quickly use much memory (4-5 Gb of total 16) and then become black and "restart".

I do not know why but guess it could have something to do with them changing the canvas implementation.

To reproduce:

1. Run the following html file in chrome 69: [ol-vector-image-layer.zip](https://github.com/openlayers/openlayers/files/2358841/ol-vector-image-layer.zip)
or use this codesandbox: https://codesandbox.io/s/9y3ypolxxr
2. Zoom with the scrollwheel back and forth for a while so the map zooms in and out. Probably a bigger window makes it happen faster. Panning also seems to make it happen.
3. Chrome should start use alot of memory suddenly after a while and then it might go black for a short time.

Tested on Win10

If testing the exact same thing on Edge everything works.
(The OSM layer also added to the testcode can be removed and still the same thing happens. It is only added to see the map zooming).

**Comment by ahocevar** [comment]
Can anyone else reproduce this? I cannot.

**Comment by AnthonNilsson** [comment]
It was actually a user also running Win10 but otherwise a completely different PC setup that reported this issue to me and then I found it happening to me 100% of the times I was testing in about 15-30 sec.

But now when I try it again I cannot either reproduce it. I am running the same tests on both my application and on the testcode I added here.

I reported this issue to Google also, maybe they have made a patch (from my issue or any other reason), but my version number in Chrome is still the same.

Anyway the issue seems to be gone, I will test it some more, but otherwise I guess it is magically solved :)

Thanks for looking into it!
